### PR TITLE
[1LP][RFR] fix python3 on travis and for --collectonly

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
.

### CODE REVIEW

File: cfme/fixtures/nelson.py [hunk 1]
```diff
@@ -28,12 +30,14 @@ def pytest_collection_modifyitems(items):
...
-        item_param = re.findall('\.*(\[.*\])', item_name)
+        item_param = re.findall(r'\.*(\[.*\])', item_name)
         if item_param:
             item_name = item_name.replace(item_param[0], '')
         node_name = '{}.{}'.format(item_class, item_name)
         output[node_name] = {}
-        output[node_name]['docstring'] = base64.b64encode(getattr(item.function, '__doc__') or '')
+        docstring = getattr(item.function, '__doc__') or ''
+        output[node_name]['docstring'] = base64.b64encode(
+            docstring if six.PY2 else docstring.encode('utf-8'))
```
[the remark this answers is not in the document]
Comment (RonnyPfannschmidt) [reply cfme/fixtures/nelson.py]: the check is needed, because `encode` will first `decode` to unicode using `ascii`

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by mshriver** [comment]
wrapanapi 2.9.1 has been tagged, please include once its available on pypi.

**Comment by mshriver** [comment]
Travis py3 build successful, and the PRT failures don't look related at all.

**Pull request merged**